### PR TITLE
git-sync: unstable-2015-10-24 -> unstable-2021-07-14

### PR DESCRIPTION
**Pull request by colonelpanic8**
…4cb3e2de10bd118f2664621ea

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/git-sync/default.nix [hunk 1]
```diff
@@ -1,15 +1,14 @@
-{ lib, stdenv, fetchFromGitHub, coreutils, gnugrep, gnused, makeWrapper, git
-}:
+{ lib, stdenv, fetchFromGitHub, coreutils, git, gnugrep, gnused, makeWrapper, inotify-tools }:
 
 stdenv.mkDerivation rec {
   pname = "git-sync";
-  version = "20151024";
+  version = "20210714";
```
Comment (prusnak) [inline pkgs/applications/version-management/git-and-tools/git-sync/default.nix:5]:
```suggestion
  version = "unstable-2021-07-14";
```

**Review by prusnak — CHANGES_REQUESTED** [review]
Change the commit message to: `git-sync: unstable-2015-10-24 -> unstable-2021-07-14`

**Review by prusnak — APPROVED** [review]

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130306` at 66fc8d23 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>git-sync</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/applications/version-management/git-and-tools/git-sync/default.nix:18:3:
```
   |
18 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/fixup-phase.md">fixup-phase</a>

`fixupPhase` should not be overridden, use `postFixup` instead.

Near pkgs/applications/version-management/git-and-tools/git-sync/default.nix:31:3:
```
   |
31 |   fixupPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/version-management/git-and-tools/git-sync/default.nix [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-  version = "20151024";
+  version = "unstable-2021-07-14";
 
   src = fetchFromGitHub {
     owner = "simonthum";
     repo = "git-sync";
-    rev = "eb9adaf2b5fd65aac1e83d6544b9076aae6af5b7";
-    sha256 = "01if8y93wa0mwbkzkzx2v1vqh47zlz4k1dysl6yh5rmppd1psknz";
+    sha256 = "sha256-PuYREW5NBkYF1tlcLTbOI8570nvHn5ifN8OIInfNNxI=";
+    rev = "7d3d34bf3ee2483fba00948f5b97f964b849a590";
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-sync/default.nix:11]:
```suggestion
    rev = "7d3d34bf3ee2483fba00948f5b97f964b849a590";
    sha256 = "sha256-PuYREW5NBkYF1tlcLTbOI8570nvHn5ifN8OIInfNNxI=";
```

**Pull request merged**